src/orquestra/quantum/api/estimator_contract.py

```diff
@@ -98,8 +99,8 @@ def _validate_expectation_value_includes_coefficients(
         estimation_tasks=estimation_tasks,
     )
 
-    return not np.array_equal(
-        expectation_values[0].values, expectation_values[1].values
+    return np.allclose(
+        expectation_values[0].values, expectation_values[1].values/19.971997, rtol = 0.1 # 10% tolerance
```

Comment:
[minor/nitpick] Could you move `19.971997` to a separate variable called, IDK, `term_coefficient`?